packages/common/src/hooks/chats/useAudienceUsers.ts

```diff
@@ -0,0 +1,52 @@
+import { ChatBlast, ChatBlastAudience } from '@audius/sdk'
+
+import {
+  useGetCurrentUserId,
+  useGetFollowers,
+  useGetPurchasers,
+  useGetRemixers,
+  useGetSupporters
+} from '~/api'
+import { UserMetadata } from '~/models'
+
+export const useAudienceUsers = (chat: ChatBlast, limit?: number) => {
+  const { data: currentUserId } = useGetCurrentUserId({})
+
+  const { data: followers } = useGetFollowers({
+    userId: currentUserId!,
+    limit
+  })
+  const { data: supporters } = useGetSupporters({
+    userId: currentUserId!,
+    limit
+  })
+  const { data: purchasers } = useGetPurchasers({
+    userId: currentUserId!,
+    contentId: parseInt(chat.audience_content_id ?? '0'),
```

Comment:
if `audience_content_id` doesn't exist, don't we want to pass `undefined` in here to get the full list of purchasers who have purchased any content by current user? 0 is still a valid content ID right?